lib/transformers/allOf.js

```diff
@@ -80,15 +74,57 @@ function allOf({schema, options, transform, data, schemaErrors} = {}) {
         });
 
         return acc;
+    }
+    if ('allOf' in sectionSchema) {
+        return sectionSchema.allOf.reduce((acc, propertiesSchema) => {
+            return transformPropertySchema(acc, propertiesSchema, acc);
+        }, {});
+    }
+    const acc = {};
+    const propertiesSchema = Object.fromEntries(
+        Object.entries(sectionSchema).filter(([key]) => key === 'properties')
+    );
+    return transformPropertySchema(acc, propertiesSchema);
+}
+
+function getPageTitle(sectionSchemaCollection) {
+    let pageHeading = sectionSchemaCollection[0].title;
+    if (pageHeading === undefined) {
+        const propertiesSchema = Object.fromEntries(
```

Comment:
as discussed, something like:

```js
if (pageHeading) { // @neil this can be done to check if something is "truthy".
    // assumed to be a length of `1`.
    const schemaProperties = Object.keys(sectionSchemaCollection[0].properties);
    pageHeading = schemaProperties[keys[0]].title;
}
return pageHeading;
```

I think `schemaProperties` is a better name of the variable here also as it is the properties of the schema being accessed here.